### ISSUE TITLE
port unblocking

### ISSUE THREAD

**Issue by FADHLyemen** [jupyterhub/binderhub#569]
When I run the below line in In my docker file:
`apt-key adv --keyserver keyserver.ubuntu.com --recv-keys EEA14886`
I got the below error in mybiner:
```
gpg: requesting key EEA14886 from hkp server keyserver.ubuntu.com
gpg: keyserver timed out
gpg: keyserver receive failed: keyserver error
```

This problem is caused by mybiner block the port.
[](https://gpgtools.tenderapp.com/kb/faq/cant-reach-key-server-are-you-behind-a-company-firewall)
I tried `apt-key adv --keyserver keyserver.ubuntu.com:80 --recv-keys EEA14886` but it does not work. 
[](https://unix.stackexchange.com/questions/75892/keyserver-timed-out-when-trying-to-add-a-gpg-public-key)
I need to send my binder link to my paper's reviewers and I appreciated any help.
[](https://github.com/FADHLyemen/lilikoi_Fadhl)

**Comment by FADHLyemen** [comment]
Yes, this is a smart solution. Thank you. 
I have a problem of the password, I read the below fix but I do not know where can I put this js file. Shall I copy it to my github page?

https://github.com/swn1/binder-build-core/blob/52c18da0e60eb9c1511620795605e17f5e7c729f/dependencies/environment-yml/index.js

**Comment by betatim** [comment]
The question about the password was fixed in gitter. The conclusion was that the link to share in this particular case is https://mybinder.org/v2/gh/FADHLyemen/lilikoi_Fadhl/master, not the one for the live binder instance which requires a password.

I will close this as your repo seems to be working. Feel free to reopen this issue if there is more on this topic or create a new issue if there is a different problem.
